### PR TITLE
Removed Event::Inline's requirement for Inline::C

### PR DESCRIPTION
**Pull request by oodler577**
Added regression test, t/Inline.t.

(note:
was not sure what to do with shebang, `# watch -*-perl-*-`)

### CODE REVIEW

File: Changes [hunk 1]
```diff
@@ -1,3 +1,8 @@
+1.27 2021-04-28
+  - removed ill advised restriction on Event::Inline, used to
+    integrate 'with' Inline::* modules
+  - *no* version bump
```
Comment (mohawk2) [inline Changes]: Just a single line with the change would have been fine. I avoid commentary about other people's code like "ill-advised" as well. You don't need to put a version header here, that's a release-manager job.

File: lib/Event.pm [hunk 2]
```diff
@@ -204,9 +204,9 @@ _load_watcher($_) for qw(idle io signal timer var);
 
 # Provide hints to Inline.pm for usage:
 # use Inline with => 'Event';
+# note: return information for all values of $language
 sub Inline {
-    my ($class, $language) = @_;
-    return if $language ne 'C'; # Inline gives good error message
```
Comment (mohawk2) [inline lib/Event.pm:209]: Please only delete this line, the note doesn't add value, nor does the change to the params line.

File: t/Inline.t [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# watch -*-perl-*-
+
+use strict;
+use Test::More tests => 4;
+use Event ();
+
+require Config;
+require Event::MakeMaker;
+
+my $path = $Event::MakeMaker::installsitearch;
```
Comment (mohawk2) [inline t/Inline.t]: This all looks very fragile. It's enough to just call the `Inline` method with `C`, and then with anything else, and check each return-value `isnt` undef.

**Review by mohawk2 — CHANGES_REQUESTED** [review]
Thanks! Just a couple of tweaks :-)

File: Changes (hunk 1 above)
Comment (oodler577) [reply Changes]: will fix

File: lib/Event.pm (hunk 2 above)
Comment (oodler577) [reply lib/Event.pm:209]: will fix

File: t/Inline.t (hunk 3 above)
Comment (oodler577) [reply t/Inline.t]: will fix

**Pull request merged**